README.md

```diff
@@ -3,9 +3,21 @@
 [![Coverage Status](https://coveralls.io/repos/OCA/${REPO_NAME}/badge.svg?branch=${BRANCH_NAME}&service=github)](https://coveralls.io/github/OCA/${REPO_NAME}?branch=${BRANCH_NAME})
```

Comment:
Please change this also to the repository name (instrastat) and the rest of needed data.